watertap/unit_models/translators/tests/test_translator_adm1_asm2d.py

```diff
@@ -283,31 +296,5 @@ def test_conservation(self, asmadm):
                     * asmadm.fs.props_ASM2D.dens_mass
                 )
             )
-            <= 1e-6
-        )
-
-        assert (
-            abs(
-                value(
-                    (
-                        asmadm.fs.unit.inlet.flow_vol[0]
-                        * asmadm.fs.props_ADM1.dens_mass
-                        * asmadm.fs.props_ADM1.cp_mass
-                        * (
-                            asmadm.fs.unit.inlet.temperature[0]
-                            - asmadm.fs.props_ADM1.temperature_ref
-                        )
-                    )
-                    - (
-                        asmadm.fs.unit.outlet.flow_vol[0]
-                        * asmadm.fs.props_ASM2D.dens_mass
-                        * asmadm.fs.props_ASM2D.cp_mass
-                        * (
-                            asmadm.fs.unit.outlet.temperature[0]
-                            - asmadm.fs.props_ASM2D.temperature_ref
-                        )
-                    )
-                )
-            )
-            <= 1e-6
```

Comment:
Is this no longer converging with <=1e-6? And is the second check just not working at all anymore? Curious whether you removed it due to test failures or because you thought it was no longer valid.